content/docs/features.md

```diff
@@ -95,38 +96,43 @@ app_running
-  {: style="color:gray; font-size: 90%"}
+  The class represents your graph, and methods in that class represents elements in that graph. 
+  Using [maven](https://maven.apache.org/), running the test with GraphWalker is done by:
+
   
   ```
   mvn graphwalker:test
   ```
   
-  GraphWalker will call the methods in the class during path generation.
-  {: style="color:gray; font-size: 90%"}
-  
-* Your test automation code has direct access to the graphs execution context.<br>{: style=""}
+  GraphWalker will call methods in the class during path generation.
 
-  When GraphWalker generates paths, it keeps all data regarding the generation in an [execution context](https://github.com/GraphWalker/graphwalker-project/blob/b604d282087db9776ebf9c4887a1224dcb642567/graphwalker-core/src/main/java/org/graphwalker/core/machine/ExecutionContext.java). Your test can query for data in your model, and it can even change data in your model.
-  {: style="color:gray; font-size: 90%"}
-  
-  An example is the [Amazon example](https://github.com/GraphWalker/graphwalker-example/tree/master/java-amazon). The code at [ShoppingCartTest.java line 65](https://github.com/GraphWalker/graphwalker-example/blob/1c66dc315fd37ca362e66704d26f194bf3acc6bd/java-amazon/src/main/java/org/graphwalker/ShoppingCartTest.java#L65) shows how the current value for the property `num_of_books` is fetched using `getAttribute` from the model.
-  {: style="color:gray; font-size: 90%"}
+* Your test automation code has direct access to the graphs execution context.  
 
+  When GraphWalker generates paths, it keeps all data regarding the generation in an 
+  [execution context](https://github.com/GraphWalker/graphwalker-project/blob/b604d282087db9776ebf9c4887a1224dcb642567/graphwalker-core/src/main/java/org/graphwalker/core/machine/ExecutionContext.java). 
+  Your test can query for data in your model, and it can even change data in your model.  
+  
+  An example is the [Amazon example](https://github.com/GraphWalker/graphwalker-example/tree/master/java-amazon). 
+  The code at [ShoppingCartTest.java line 65](https://github.com/GraphWalker/graphwalker-example/blob/1c66dc315fd37ca362e66704d26f194bf3acc6bd/java-amazon/src/main/java/org/graphwalker/ShoppingCartTest.java#L65) shows how the current value for the property `num_of_books` is fetched using `getAttribute` from the model.
+  
 ## What GraphWalker does not do
 
-### Test execution
+### Management of the system under test
 
-GraphWalker does not interact with your system under test. You need some other tool do that. If, for instance, you want to test a web application, you would perhaps use [Selenium](http://www.seleniumhq.org/) to do that. Or, if your target is a mobile app, then [Appium](http://appium.io/) might be you choice.
+GraphWalker does not interact with your system under test. You need some other tool do that. 
```

Comment:
This is not really a change, is it? You added some new lines, but the mark down rendering is still the same, right?
If so, I suggest not to make a change here.